File: Python/EstruturaDeDecisao/exercicio_07.py
"""
Faça um Programa que leia três números e mostre o maior e o menor deles.
"""

num1 = float(input('Digite o primeiro número: '))
num2 = float(input('Digite o segundo número: '))
num3 = float(input('Digite o terceiro número: '))

maior = max(num1, num2, num3)
menor = min(num1, num2, num3)
# OU
if num1 >= (num2) and (num3):
    maior = num1
elif num2 >= (num1) and (num3):
    maior = num2
else:
    maior = num3

if num1 <= (num2) and (num3):
    menor = num1
elif num2 <= (num1) and (num3):
    menor = num2
else:
    menor = num3

print(f'O maior número é: {maior}')
print(f'O menor número é: {menor}')

File: Java/EstruturaDeDecisao/Exercicio07.java
// Faça um Programa que leia três números e mostre o maior e o menor deles.
package EstruturaDeDecisao;
import java.util.Scanner;

public class Exercicio07 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double numero1 = scanner.nextDouble();
        
        System.out.print("Digite o segundo número: ");
        double numero2 = scanner.nextDouble();

        System.out.print("Digite o terceiro número: ");
        double numero3 = scanner.nextDouble();

        // Encontrar o maior número entre os três
        double maiorNumero = numero1;
        if (numero2 > maiorNumero) {
            maiorNumero = numero2;
        }
        if (numero3 > maiorNumero) {
            maiorNumero = numero3;
        }

        // Encontrar o menor número entre os três
        double menorNumero = numero1;
        if (numero2 < menorNumero) {
            menorNumero = numero2;
        }
        if (numero3 < menorNumero) {
            menorNumero = numero3;
        }

        System.out.println("O maior número é: " + maiorNumero);
        System.out.println("O menor número é: " + menorNumero);

        scanner.close();
    }
}
